packages/exchange/src/exchange/providers/ollama.py

```diff
@@ -16,14 +16,14 @@ class OllamaProvider(OpenAiProvider):
 
 Here's an example profile configuration to try:
 
-    ollama:
-      provider: ollama
-      processor: {OLLAMA_MODEL}
-      accelerator: {OLLAMA_MODEL}
-      moderator: passive
-      toolkits:
-      - name: developer
-        requires: {{}}
+ollama:
+  provider: ollama
+  processor: qwen2
```

Comment:
qwen2.5? Also should we not change OLLAMA_MODEL to that?

Reply:
testing it now

Reply:
@codefromthecrypt will it change CI behavior? so far qwen 2.5 seems quite good

Reply:
sorry I was AWOL last week. CI tests with qwen2.5, so will follow-up with a change to update the constant

Reply:
https://github.com/block-open-source/goose/pull/150